### PR TITLE
three more usage examples added to readme(#5)

### LINKED ISSUE #5: Add more usage in README
Add some more examples of usage in readme usage tab.

### PR DESCRIPTION
**Pull request by DavidLeeR**
Fixes #5 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ print(dollarSymbol)
 
 # Output:
 # $
+
+euroSymbol = CurrencySymbols.get_symbol('EUR');
+print(dollarSymbol)
```
Comment (arshadkazmi42) [inline README.md]: This should be `print(euroSymbol)`

File: README.md [hunk 2]
```diff
@@ -25,6 +25,24 @@ print(dollarSymbol)
 
 # Output:
 # $
+
+euroSymbol = CurrencySymbols.get_symbol('EUR');
```
Comment (arshadkazmi42) [inline README.md]: Remove semicolon `;`

File: README.md [hunk 3]
```diff
@@ -25,6 +25,24 @@ print(dollarSymbol)
...
+print(dollarSymbol)
+
+# Output:
+# €
+
+britishPoundSymbol = CurrencySymbols.get_symbol('EUR');
```
Comment (arshadkazmi42) [inline README.md]: remove semicolon

Comment (arshadkazmi42) [reply README.md]: this should be `CurrencySymbols.get_symbol('GBP')`

File: README.md [hunk 4]
```diff
@@ -25,6 +25,24 @@ print(dollarSymbol)
...
+print(britishPoundSymbol)
+
+# Output:
+# £
+
+bitcoinSymbol = CurrencySymbols.get_symbol('BTC');
```
Comment (arshadkazmi42) [inline README.md]: remove semicolon

**Comment by arshadkazmi42** [comment]
Great work. Thank  you.

**Comment by DavidLeeR** [comment]
Your very welcome!

**Pull request merged**